interfaces/builtin/hidraw_test.go

```diff
@@ -169,12 +193,15 @@ func (s *HidrawInterfaceSuite) TestSanitizeBadCoreSnapSlots(c *C) {
 func (s *HidrawInterfaceSuite) TestSanitizeGadgetSnapSlots(c *C) {
 	c.Assert(s.testUDev1.Sanitize(s.iface), IsNil)
 	c.Assert(s.testUDev2.Sanitize(s.iface), IsNil)
+	c.Assert(s.testUDev3.Sanitize(s.iface), IsNil)
 }
 
 func (s *HidrawInterfaceSuite) TestSanitizeBadGadgetSnapSlots(c *C) {
 	c.Assert(s.testUDevBadValue1.Sanitize(s.iface), ErrorMatches, "hidraw usb-vendor attribute not valid: -1")
 	c.Assert(s.testUDevBadValue2.Sanitize(s.iface), ErrorMatches, "hidraw usb-product attribute not valid: 65536")
 	c.Assert(s.testUDevBadValue3.Sanitize(s.iface), ErrorMatches, "hidraw path attribute specifies invalid symlink location")
+	c.Assert(s.testUDevBadValue4.Sanitize(s.iface), ErrorMatches, "hidraw usb-interface-number attribute not valid: -1")
+	c.Assert(s.testUDevBadValue5.Sanitize(s.iface), ErrorMatches, "hidraw usb-interface-number attribute not valid: 33")
```

Comment:
It would be great to say why the number is not valid (`usb-interface-number cannot be negative`, `usb-interface-number cannot be larger than XX`)

Reply:
Ok, I'll use "usb-interface-number attribute cannot be negative and larger than 31"